tests/sdk/sdk_integration_test.py

```
@@ -0,0 +1,106 @@
+    """
+    http_client = httpx.Client(
+        transport=client._transport,  # pyright: ignore
+        base_url=str(client.base_url),
+        headers=client.headers,
+    )
+
+    honcho_client = Honcho(workspace_id="sdk-test-workspace", http_client=http_client)
+    yield honcho_client
+
+
+def test_peer_operations(honcho_test_client: Honcho):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_peer_operations`

Add return type annotation: `None`

(ANN201)